dcpy/utils/postgres.py

```diff
@@ -153,6 +171,19 @@ def get_table_columns(self, table_name: str) -> list[str]:
         )
         return sorted(column_names["column_name"])
 
+    def add_table_column(
+        self,
+        table_name: str,
+        *,
+        col_name: str,
+        col_type: str,
+        default_value: str = "",
+    ):
+        q = f"ALTER TABLE {table_name} ADD COLUMN {col_name} {col_type}"
+        if default_value:
```

Comment:
should this be `if default_value == ""`? doesn't seem like it'd ever be `None` yet

Reply:
Well, this being python, you never really know what you'll get, so I opted for the `truthiness` approach. Happy to change if you feel strongly.

Reply:
o yea I forgot `if ""` evaluates to `False` !